muon/browser/muon_browser_process_impl.cc

```diff
@@ -19,6 +20,8 @@ MuonBrowserProcessImpl::MuonBrowserProcessImpl(
       const base::CommandLine& command_line) :
     BrowserProcessImpl(local_state_task_runner, command_line) {
   g_browser_process = this;
+
+  device_client_.reset(new ChromeDeviceClient);
```

Comment:
this should actually go in chromium_src browser_process_impl.cc
the muon subclass is for muon-specific stuff